flow/flow.go

```diff
@@ -496,8 +513,8 @@ func SetReader(filename string, repcount int32) (OUT *Flow) {
 func SetReceiver(par interface{}) (OUT *Flow) {
 	ring := low.CreateRing(generateRingName(), burstSize*sizeMultiplier)
 	var recv *scheduler.FlowFunction
-	if port, t := par.(uint8); t {
-		if port >= uint8(len(createdPorts)) {
+	if port, t := par.(int); t {
```

Comment:
If int here, also need to modify files examples/Firewall.go and other - remove port conversion to uint8